docs/DeveloperGuide.md

```diff
@@ -609,28 +610,293 @@ To sum it up, this application helps users to reduce the time needed to transfor
 
 - **Mainstream OS**: Windows, Linux, Unix, OS-X
 
-## Appendix E: Instructions for Manual Testing
+**Note:** These instructions only provide a starting point for testers to work on, testers are expected to do more *exploratory* testing.
 
-### Start-up and Shutdown
+### Start and Shutdown
+
+1. Ensure you have Java version `11` or above installed on your computer.
+2. Next, download the latest `decodex.jar` [here](https://github.com/AY2122S1-CS2113T-T10-3/tp/releases).
+3. Copy the `decodex.jar` into an empty folder.
+4. Run the `decodex.jar` on your command prompt by typing in `java -jar decodex.jar` the press `Enter`.
+5. Type a command in the command box after the `Decodex [] >` prompt and press Enter to execute it. e.g. typing `help` and pressing `Enter` will show a list available commands, their respective descriptions and syntaxes.
+6. To exit Decodex, use the `exit` command. Alternatively, Decodex can be exited using `Ctrl+C` or by closing the command prompt window.
 
 ### Inputting Data
 
+When entering data, `Decodex` requires a specific format such that the data is entered correctly.
+
+<aside>
+ℹ️ - Note that the program will consider **all characters as data following the first space character** in the input command. You may observe leading and trailing whitespaces in the output.
```

Comment:
perhaps "all characters as data following the first space character" can be changed to "all characters following the first space character as data"?